modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new `tablecellborderstyle` toolbar button and menu item to change table cell border style #TINY-7478
 - Added a new `tablecaption` toolbar button and menu item to toggle captions on tables #TINY-7479
 - Added a new `mceTableToggleCaption` command that toggles captions on a selected table #TINY-7479
+- Added a new `language` menu item and toolbar button to set the `lang` attribute on content #TINY-6149
+- Added a new `content_langs` setting to specify the languages available for the `language` menu item and toolbar button #TINY-6149
```

Comment:
This is all part of the one feature and has duplicate information, so please merge this into a single changelog entry.

- [ ] Merge into a single changelog entry

Reply:
Merged, as per discussion in slack. Would be nice to have some more opinions on whether the nested entry is good or not.

Reply:
We did discuss the other day already, but it seems you may have forgotten about it? I've added a suggestion anyways.